File: frontend/src/model/Booking.js
export function Booking(number, account, activity,
    active, completed, pending, modifiedBy, createdBy,
    modificationDate, creationDate) {

  this.number = number;
  this.account = account;
  this.activity = activity;
  this.active = active;
  this.completed = completed;
  this.pending = pending;
  this.modifiedBy = modifiedBy;
  this.createdBy = createdBy;
  this.creationDate = creationDate;
  this.modificationDate = modificationDate;
}

File: backend/core/domain/src/main/java/pl/lodz/p/it/core/domain/Booking.java
package pl.lodz.p.it.core.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.SuperBuilder;
import pl.lodz.p.it.core.shared.validation.BookingNumber;

/**
 * Class responsible for keeping a domain model of the booking object.
 */
@EqualsAndHashCode(callSuper = true)
@Data
@SuperBuilder
@AllArgsConstructor
public class Booking extends BaseModel {

    @BookingNumber
    private String number;

    private String account;

    private String activity;

    private Boolean active;

    private Boolean completed;

    private Boolean pending;

    private String modifiedBy;

    private String createdBy;
}
